### ISSUE TITLE
App crashes randomly when failing to merge settings

### ISSUE THREAD

**Issue by Allen1351** [files-community/Files#8601]
### Description

The app keeps crashing, no stable patron, just in the first 25 seconds of being opened

### Steps To Reproduce

1. Mi PC
2.  open disk D
3. Open any folder
4. Crash

### Expected behavior

not crashing

### Files Version

Version: 2.1.15.0 OS Version: 10.0.19044.1526

### Windows Version

Edition	Windows 10 Pro Version	21H2 Installed on	‎11/‎30/‎2021 OS build	19044.1526 Experience	Windows Feature Experience Pack 120.2212.4170.0

### Relevant Assets/Logs

[debug_fulltrust.log](https://github.com/files-community/Files/files/8203939/debug_fulltrust.log)
[debug.log](https://github.com/files-community/Files/files/8203941/debug.log)

**Comment by Josh-65** [comment]
Can you try reinstalling

**Comment by Josh-65** [comment]
Is this still an issue in 2.3.11.0 or above?

**Comment by Allen1351** [comment]
In 2.3.11 I have not experienced any crashes so far. But the issue with PDF thumbnails still persists. I really wish to solve this. It's preventing me from using the app since I need to see the content of the pdf to open the ones I need quickly. I installed this version on my work PC, which has Acrobat instead of Foxit and the thumbnails are still missing. Again, they show fine in Windows Explorer. I tried resetting the thumbnail generation app by default after installing it to see if it solved the issue but still nothing. It displays perfectly fine jpg thumbnails, even DWG thumbnails, but no pdf.
